### PR TITLE
Remove translation file if all translations removed

### PR DESCRIPTION
**Pull request by michealmouner**
bug:
once all translations removed the file not deleted

I deleted all files that not exist in the generated domains anymore

### CODE REVIEW

File: Translation/Updater.php [hunk 1]
```diff
@@ -177,6 +177,21 @@ public function process(Config $config)
...
+        $endOfFile = sprintf('.%s.%s', $this->config->getLocale(), $format);
+        $translationFilesRegex = sprintf('/%s$/', $endOfFile);
+        foreach(Finder::create()->name($translationFilesRegex)->in($this->config->getTranslationsDir())->files() as $file){
+            $domainName = str_replace($endOfFile, '', $file->getFilename());
+            if($this->scannedCatalogue->hasDomain($domainName)){
+                continue;
+            }
+
+            $this->logger->info(sprintf('Deleting translation file "%s".', $file));
+            if (false === @unlink((string) $file)) {
```
Comment (TamasSzigeti) [inline Translation/Updater.php]: Why is the `@` needed?

Comment (michealmouner) [reply Translation/Updater.php]: I used the same logic above for deleting a file (just for consistency)

Comment (TamasSzigeti) [reply Translation/Updater.php]: Can it ever not exist?

Comment (michealmouner) [reply Translation/Updater.php]: 🤔  🤔  🤔  🤔  
no, I will remove it

**Review by TamasSzigeti — APPROVED** [review]

**Pull request merged**